db/migrate/20180404150319_create_reports.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReports < ActiveRecord::Migration[5.0]
+  def change
+    create_table :reports do |t|
+      t.json :data
+      t.integer :quarter
+      t.references :user, foreign_key: true
+      t.integer :school_id
```

Comment:
Let also use `references` for this also

Reply:
@sofiengwin all issues have been addressed.